todo_app/templates/index.html

```diff
@@ -11,8 +11,16 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
```

Comment:
Commenting on the wrong file as GitHub won't let me comment on an empty file - but worth noting that it looks like you've accidentally checked in an empty `index` file in this folder